pkg/auth/model/model.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+option go_package = "github.com/treeverse/lakefs/auth/model";
+
+import "google/protobuf/timestamp.proto";
+
+package io.treeverse.lakefs.auth.model;
+
+message UserData {
```

Comment:
The convention in proto files is CamelCase for messages and snake_case for variable names